src/foam/u2/crunch/PermissionsStringArrayView.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch',
+  name: 'PermissionsStringArrayView',
+  extends: 'foam.u2.View',
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  imports: [
+    'permissionDAO'
+  ],
+
+  requires: [
+    'foam.nanos.auth.Permission',
+    'foam.u2.crunch.PermissionSelection',
+    'foam.u2.TextField'
+  ],
+
+  properties: [
+    {
+      name: 'search',
+      class: 'String',
+      view: {
+        class: 'foam.u2.TextField',
+        type: 'search',
+        placeholder: 'capability search',
+        onKey: true
+      },
+      readVisibility: foam.u2.DisplayMode.RW,//would like it to be accessible even in DAOSummaryView to serch for permissions, but is there any legit way to do that?
+    },
+    {
+      name: 'views',
+      class: 'Array',
+      of: 'foam.u2.crunch.PermissionSelection',
+      factory: function() {
+        var self = this;
+        this.permissionDAO.select(this.PROJECTION(this.Permission.ID))
+          .then(function(proj) {
+            self.preSetViewWithProjection(proj);
+          });
+      }
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+
+      this.search$.sub(function() {
```

Comment:
Easier/faster to just display all permissions, then have each one listen for search and hide itself if it doesn't match.